### PR TITLE
feature/add-gitmoji-in-front-of-merge-commit-title

### LINKED ISSUE #73: ✨ Automatically add 🔀 in front of merge commit title input in GitHub/GitLab
Hello Team 👋

**Describe the feature that you want see**

In a PR, GitHub can automatically merge your branch with a generated message:
image

I could be really useful that the extension detect it and add the 🔀 or `:twisted_rightwards_arrows:` gitmoji in front of the message.

### PR DESCRIPTION
**Pull request by johannchopin**
Close #73

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -20,7 +20,7 @@
   "rules": {
     "eol-last": ["error", "always"],
     "indent": ["warn", 2],
-    "max-len": ["error", { "code": 95, "ignoreComments": true }],
```
Comment (Lyokolux) [inline .eslintrc:23]: Why ?

File: src/pages/Settings.svelte [hunk 2]
```diff
@@ -70,7 +71,7 @@
 </header>
 
 <div class="settings">
-  <div class="setting theme">
+  <div class="setting center">
```
Comment (Lyokolux) [inline src/pages/Settings.svelte:74]: Nice catch !

**Review by Lyokolux — COMMENTED** [review]
In `src/components/ToggleButton.svelte` (l.64), there is a `<label>` tag that is not linked to its input-

> A11y: A form label must be associated with a control

File: .eslintrc (hunk 1 above)
Comment (johannchopin) [reply .eslintrc:23]: To write longer code line like [this one](https://github.com/johannchopin/gitmoji-browser-extension/pull/74/files#diff-52b3e2692dcfb066a0d97b2585d3e6e7f1674ac744449e8e1e75709d5cb044b2R44)

**Comment by johannchopin** [comment]
@Lyokolux The label here is just for the UI rendering so it's not a real issue 👍 I just ignored it. Feel free to re-review it and validate the PR ✅

File: .eslintrc (hunk 1 above)
Comment (Lyokolux) [reply .eslintrc:23]: :white_check_mark: Valid example :+1:

**Review by Lyokolux — APPROVED** [review]
All threads are resolved.

**Pull request merged**